src/components/views/settings/tabs/user/AppearanceUserSettingsTab.tsx

```diff
@@ -296,8 +298,9 @@ export default class AppearanceUserSettingsTab extends React.Component<IProps, I
     }
 
     private renderFontSection() {
-        return (SettingsStore.getValue(UIFeature.ChangeFont)) &&
-        <div className="mx_SettingsTab_section mx_AppearanceUserSettingsTab_fontScaling">
+        const defaultFontSize = SettingsStore.getValue(UIFeature.ShowDefaultFontSize) || 10;
```

Comment:
As this config setting is not controlling if something is shown or hidden, I suggest the name should be DefaultFontSize

Reply:
No worries. I have implemented the change.